client/src/contexts/ItemsContext.jsx

```diff
@@ -138,12 +140,27 @@ export function useItems() {
 
 // Provides the store to your application
 export function ItemsProvider({ children }) {
+  const [items, setItems] = useState('')
```

Comment:
Nice work so far Anderson! Very nice to see you connect the front end and back end of the app.
Since items is an array it might be better to have the default state be an empty array.
So `useState([])` instead of `useState('')`